daphne/src/metrics.rs

```diff
@@ -120,3 +115,11 @@ pub enum DaphneRequestType {
     /// DAP collect request.
     Collect,
 }
+
+#[derive(Clone, Copy, Debug)]
+pub enum DaphneAggregationJobStatus {
+    /// AggregationJobInitReq
+    Started,
+    /// AggregationJobContinueReq
```

Comment:
I would avoid leaking protocol details here, since they are subject to change over time and somewhat orthogonal to these variants (at least at a conceptual level).
```suggestion
    Started,
```